pkg/proc/native/proc.go

```diff
@@ -339,10 +339,12 @@ func (dbp *Process) SwitchGoroutine(gid int) error {
 }
 
 // FindBreakpoint finds the breakpoint for the given pc.
-func (dbp *Process) FindBreakpoint(pc uint64) (*proc.Breakpoint, bool) {
-	// Check to see if address is past the breakpoint, (i.e. breakpoint was hit).
-	if bp, ok := dbp.breakpoints.M[pc-uint64(dbp.bi.Arch.BreakpointSize())]; ok {
-		return bp, true
+func (dbp *Process) FindBreakpoint(pc uint64, afterHit bool) (*proc.Breakpoint, bool) {
+	if afterHit {
```

Comment:
nit: I could be convinced otherwise, but I almost feel like `adjustPC` might be a clearer name.